dragula.js

```diff
@@ -155,11 +161,11 @@ function dragula (initialContainers, options) {
     if (drake.dragging && _mirror) {
       return;
     }
-    if (isContainer(item)) {
+    var handle = item;
+    if (isContainer(item, handle) && !o.allowNestedContainers) {
```

Comment:
`var handle = item;`

What's the point of linking `item` into a new variable? You could as well just use `item` directly, as it's precisely the same thing.

If you need to create a new object with the same contents, but also avoid linking and possibly mutating the original object, you can do as such:

```
var handle = Object.assign({}, item);
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill

Cheers